src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,44 @@
+package step1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+    @Test
+    void test1(){
+        assertThat(numbers.size()).isEqualTo(3);

```

Comment:
```suggestion
assertThat(numbers).hasSize(3);
```

이런식으로도 작성이 가능합니다. 좀 더 직관적일수도 있을거 같아요. assertJ는 여러가지 메서드들을 제공해줍니다.